src/components/features/ClientFeedback/ClientFeedback.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './ClientFeedback.module.scss';
+import Feedback from '../../common/Feedback/FeedbackContainer';
+
+class ClientFeedback extends React.Component {
+  state = {
+    activePage: 0,
+    activeCategory: 'Furniture client',
+  };
+
+  handlePageChange(newPage) {
+    this.setState({ activePage: newPage });
+  }
+
+  handleCategoryChange(newCategory) {
+    this.setState({ activeCategory: newCategory });
+  }
+
+  render() {
+    const { products, viewport } = this.props;
+    const { activeCategory, activePage } = this.state;
+
+    const itemsPerViewport = { desktop: 1, tablet: 1, mobile: 1 };
+    const itemsPerPage = itemsPerViewport[viewport];
+
+    const colsPerViewport = { desktop: 12, tablet: 6, mobile: 1 };
+    const colsOnPage = colsPerViewport[viewport];
+
+    const categoryProducts = products.filter(item => item.category === activeCategory);
```

Comment:
Po raz kolejny piszę, żę tutaj nie potrzebujemy tych linii - zostąło to skopiowane z innego komponenetu gdzie był kategorie, różne formy prezentacji w zależnosći od vieport, tutaj tego nie ma prezentujemy jedna opinię na jednej stronie.